### DISCUSSION TITLE
Add possibility to return the linear transform in align_with_landmarks

### DISCUSSION THREAD

**Discussion by baba-yaga** [marcomusy/vedo#1014] [Ideas]
It would be nice to have a possibility to return the linear transform by `align_with_landmarks` function in `pointcloud.py`. It may then be re-used with `.apply_transform` on other objects too. For instance, I fit a mesh to an image `img` of its rotated view, then apply the inverse transform to the camera showing `Image(img)` to find the position of the camera in the original mesh space which gives the same view as on `img`. I am going to describe this in Discussion soon.

Technically, I suggest adding argument `return_transform=False` in the function call and return `lmt` at coming to the last 3 lines of the code if it is `True`.

I wonder if the non-linear transform can be returned from `warp` function too. It can then be saved and applied later.

**Comment by marcomusy** [comment]
Hey Sergei, this is already possible for both linear and non-linear operations!
Check out example: `examples/basic/align5.py`. Try adding 
```
print(s2.transform)
```

will print:
```
vedo.transformations.LinearTransform at (0x7f4790b453d0)           
name          : LinearTransform
concatenations: 1
inverse flag  : False
matrix 4x4    :
[[ 1.781236, -0.576785,  0.051535,  0.344465],
 [ 0.577714,  1.758522, -0.286335, -0.019739],
 [ 0.039791,  0.288202,  1.850269, -0.026763],
 [ 0.      ,  0.      ,  0.      ,  1.      ]]
```

this is a `vedo.LinearTransform` which you can later use with `some_other_mesh.apply_transform(T)`

**Reply by baba-yaga** [reply]
Indeed! Sorry for missing it - thank you!
